app/controllers/api/v1/sops_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api::V1
+  class SopsController < ApiController
+    def index
+      keyword, from, size = sop_search_params
+      @search = Sop.search({ keyword: keyword, from: from, size: size }).results.results
+      respond_to do |format|
+        format.json { render json: @search }
+      end
+    end
+
+    private
+
+    def sop_search_params
+      params.values_at(:searchText, :from, :size)
```

Comment:
consider renaming client request to pass :keyword instead of :searchText and use params.permit(:keyword, :from, :size)

Reply:
good idea 👍